airflow/migrations/versions/22ab4efd5674_add_token_blocklist.py

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add-token-blocklist
+
+Revision ID: 22ab4efd5674
+Revises: a13f7613ad25
+Create Date: 2021-04-17 18:16:31.019394
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '22ab4efd5674'
+down_revision = 'a13f7613ad25'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Apply Add token blocklist table"""
+    op.create_table(
+        "token_blocklist",
+        sa.Column("id", sa.Integer(), primary_key=True),
+        sa.Column("jti", sa.String(50), nullable=False, unique=True),
```

Comment:
```suggestion
        sa.Column("jti", sa.String(50), nullable=False, primary_key=True),
```

If we use the `jti` as the primary key then the queries get simpler -- there's no need for an auto-increment/integer primary key for the DB, and the jti is unique, so lets just use that.

Doing this makes the get query easier too:

```python
    session.query(TokenBlockList).get(jti)
```